containers/unit_tests/TestDualView.hpp

```diff
@@ -622,6 +622,22 @@ TEST(TEST_CATEGORY,
             const_dv.view<Kokkos::DefaultExecutionSpace>());
 }
 
+TEST(TEST_CATEGORY, dualview_of_view) {
+  Kokkos::DualView<Kokkos::View<double*, TEST_EXECSPACE>*, TEST_EXECSPACE> dv_v(
+      Kokkos::view_alloc("myView", Kokkos::SequentialHostInit), 3u);
+
+  Kokkos::View<double*, TEST_EXECSPACE> v("v", 2);
+  Kokkos::View<double*, TEST_EXECSPACE> w("w", 2);
+  dv_v.h_view(0) = v;
+  dv_v.h_view(1) = w;
+
+  dv_v.modify_host();
+  dv_v.sync_device();
+
+  dv_v.resize(Kokkos::view_alloc(Kokkos::SequentialHostInit), 2u);
+  ASSERT_EQ(dv_v.d_view.size(), 2u);
+  ASSERT_EQ(dv_v.h_view.size(), 2u);
+}
```

Comment:
What about realloc?

Reply:
> What about realloc?

See pull requets description. In more detail: it's broken anyway for View of View since we are doing `Kokkos::deep_copy(v, v::value_type{})` which isn't interpreted as "copy a scalar to a View" but as "copy between two Views"

Reply:
Can we test realloc without view of views?